GDJS/Runtime/runtimescene.ts

```diff
@@ -560,12 +560,14 @@ namespace gdjs {
      * Visibility is set to false if object is hidden, or if
      * object is too far from the camera of its layer ("culling").
      */
-    _updateObjectsVisibility() {
+    _updateObjectsPreRender() {
       if (this._timeManager.isFirstFrame()) {
         this._constructListOfAllInstances();
         for (let i = 0, len = this._allInstancesList.length; i < len; ++i) {
-          let object = this._allInstancesList[i];
-          let rendererObject = object.getRendererObject();
+          const object = this._allInstancesList[i];
+          // Perform pre-render update.
+          object.updatePreRender(this);
```

Comment:
Maybe we should do it this after the `visible` update? Not sure tbh

Reply:
I'm not sure about that as well 🤔.

Reply:
Sure!